### DISCUSSION TITLE
Binding on event

### DISCUSSION THREAD

**Discussion by FournyP** [AvaloniaUI/Avalonia#5723] [Q&A]
Hello everyone, I have created a issue but I think it is more suitable to post my question here.

Link to my issue : https://github.com/AvaloniaUI/Avalonia/issues/5721

I'm currently working on a Avalonia application and I have some problem

I try to bind a function on Gestures.DoubleTapped event but an error occured.

Here my code :

```
<DataGrid 
		x:Name="dataGrid"
		Margin="10"
		AutoGenerateColumns="False" 
		Items="{Binding Files}"
		CanUserReorderColumns="True"
		CanUserResizeColumns="True"
		CanUserSortColumns="True"
		IsReadOnly="True" 
		SelectionMode="Single"
		Gestures.DoubleTapped="{Binding Download}">
		<DataGrid.Columns>
			<DataGridTextColumn 
				Header="Name" 
				Binding="{Binding Name}" />
			<DataGridTextColumn 
				Header="Last Update" 
				Binding="{Binding LastModifyingDatetime}" />
			<DataGridTextColumn 
				Header="Type"
				Binding="{Binding Extension}" />
			<DataGridTextColumn 
				Header="Size"
				Binding="{Binding Size}" />
		</DataGrid.Columns>
	</DataGrid>
```

And the error is : 
![image](https://user-images.githubusercontent.com/64586968/113117949-41bcf780-920f-11eb-9278-d958bd59323d.png)

I have searched here in the doc : https://avaloniaui.net/docs/input/events#attached-events-in-avalonia, but I didn't find how to fix the error.

Have a good day :)

**Comment by workgroupengineering** [comment]
Gestures.Double Tapped does not support ICommand only EventHandler

To Map Event to Command you can use https://github.com/wieslawsoltes/AvaloniaBehaviors
